2020-09-08 03:47:32,996 ERROR 13372 nodejs.ClusterClientNoResponseError: client no response in 10800279ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (D:\energy-platform-project\energy-platform-server\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 13372
hostname: J9MO9BU01VTBF92

2020-09-08 08:11:55,342 ERROR 13372 nodejs.ClusterClientNoResponseError: client no response in 15861921ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (D:\energy-platform-project\energy-platform-server\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 13372
hostname: J9MO9BU01VTBF92

2020-09-08 08:34:54,852 ERROR 13372 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:205:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 13372
hostname: J9MO9BU01VTBF92

2020-09-08 09:18:47,872 ERROR 15188 nodejs.TypeError: Cannot read property 'code' of null (uncaughtException throw 1 times on pid:15188)
    at Timeout._onTimeout (D:\energy-platform-project\energy-platform-server\app\controller\mail.js:27:41)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)

pid: 15188
hostname: J9MO9BU01VTBF92

2020-09-08 10:01:28,718 ERROR 13372 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:205:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 13372
hostname: J9MO9BU01VTBF92

2020-09-08 10:22:12,391 ERROR 4816 [-/127.0.0.1/-/36ms POST /user/update] nodejs.ReferenceError: _data is not defined
    at UserService.updateuser (D:\energy-platform-project\energy-platform-server\app\service\user.js:17:21)
    at UsreController.updateuser (D:\energy-platform-project\energy-platform-server\app\controller\user.js:72:51)
    at Object.callFn (D:\energy-platform-project\energy-platform-server\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\energy-platform-project\energy-platform-server\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\energy-platform-project\energy-platform-server\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\energy-platform-project\energy-platform-server\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\energy-platform-project\energy-platform-server\node_modules\koa-compose\index.js:44:32)
    at next (D:\energy-platform-project\energy-platform-server\node_modules\koa-compose\index.js:45:18)
    at D:\energy-platform-project\energy-platform-server\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\energy-platform-project\energy-platform-server\node_modules\koa-compose\index.js:44:32)

pid: 4816
hostname: J9MO9BU01VTBF92

2020-09-08 10:22:27,008 ERROR 720 [-/127.0.0.1/-/42ms POST /user/update] nodejs.ReferenceError: _ata is not defined
    at UserService.updateuser (D:\energy-platform-project\energy-platform-server\app\service\user.js:17:21)
    at UsreController.updateuser (D:\energy-platform-project\energy-platform-server\app\controller\user.js:72:51)
    at Object.callFn (D:\energy-platform-project\energy-platform-server\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\energy-platform-project\energy-platform-server\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\energy-platform-project\energy-platform-server\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\energy-platform-project\energy-platform-server\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\energy-platform-project\energy-platform-server\node_modules\koa-compose\index.js:44:32)
    at next (D:\energy-platform-project\energy-platform-server\node_modules\koa-compose\index.js:45:18)
    at D:\energy-platform-project\energy-platform-server\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\energy-platform-project\energy-platform-server\node_modules\koa-compose\index.js:44:32)

pid: 720
hostname: J9MO9BU01VTBF92

2020-09-08 21:08:41,148 ERROR 13372 nodejs.ClusterClientNoResponseError: client no response in 10816509ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (D:\energy-platform-project\energy-platform-server\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 13372
hostname: J9MO9BU01VTBF92

2020-09-09 08:20:17,590 ERROR 13372 nodejs.ClusterClientNoResponseError: client no response in 40295896ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (D:\energy-platform-project\energy-platform-server\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 13372
hostname: J9MO9BU01VTBF92

